CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change log
 
 - The opsets ``ai.onnx`` version 20 and ``ai.onnx.ml`` version 4 (ONNX 1.15) are now shipped with Spox.
 
+**Other changes**
+
+- Internal spox classes of node attributes now receive the attribute's name earlier, in the constructor, instead of passing it later when building the attribute proto.
+
```

Comment:
Unsure what others think - but I don't think this is important in a changelog. It does have a place in the commit history and that's probably enough, as it should just be an internal refactor.

Reply:
I agree, the changelog should document the impact for the user. This can be rolled into improved caching/performance once merged to the main branch.